### PR TITLE
ci: windows travis build

### PR DESCRIPTION
**Pull request by OzoneGrif**

### CODE REVIEW

File: compiler/stack.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-15.9
+resolver: lts-15.3
```
Comment (pacien) [inline compiler/stack.yaml:20]: Please add a comment with a link to the revelant issue forcing the downgrade.

File: .travis.yml [hunk 2]
```diff
@@ -56,3 +57,66 @@ jobs:
...
+        secure: zRJV96jl3zchT88YAtT6irCXgm96qVcIokVmCHrzcAXsRHgTHQURKcdpPzFdZQnNkxwfS1akA3fW5icr+U6WZIxIe56sHYEaBg7VXR1RiGW/qNOAhSvbJGva75iUllTiELa2FL2gEb8bNhVOy3+bQr5Z7RMeHFiC4FFmbMvaQhd5ss3yR5+bqKufEuJPDpFap93XHUoczXH+lXp1v90Cia1gskHsSl+YgfGWQAc89yOxETwFOSqr3CScZGV6oJpGM2rsflN5arFS/8JqDwuemttfrt74wDdKR4Mk94tcqXqMrledUsXR1NEjWAEWW6QMj05ztZ76TkT2hKpG+WVmUTaJjNKS+RJnf4WKzv1vo2EHZuAuPvwR21NGVYTxvV4o3Zvs4YODGXiUxQgeF5LFA3jbZw9ODyloStUV7zIPqzL7qJEIehnMtkjo2JPav5ORz6B7GwVrDC4LJHn95on3/3Voo3mPeyepoz1gFoh3iovTKAc+IXQXGOhT7cATP9DAWLO/Epct7hWPCEflm+oSS4rNiVbMW/61O3yIpxgkJ/oTsixLk6LOhWNdr2hfP95nAtD+It7LSsJABmWJ0FVO9RtQC7fLmEPaoGdZIdeRk/0yig9vA78Y+q46B6LBalmZO+0V60rdLdrnggJsyt3DZKM/4Z+QUwe494NodE7hU9U=
+      draft: true
+      on:
+        repo: pacien/ldgallery
+        branch:
+        - master
+        - staging
+      overwrite: true
+      cleanup: false
+      file: ldgallery-win64.7z
```
Comment (pacien) [inline .travis.yml:81]: `.zip` here.

File: .travis.yml [hunk 3]
```diff
@@ -56,3 +57,66 @@ jobs:
...
+      - NVM_HOME="$ProgramData/nvm/"
+      - NVM_SYMLINK="$ProgramData/nvm/nodejs/"
+      - STACK_ROOT="$ProgramData/haskell/"
+
+    before_install:
+    - export "PATH=$NVM_HOME:$NVM_SYMLINK:$STACK_ROOT:$PATH"
+    - echo $PATH
+    - choco install nvm
+    - nvm install 12.16.2
+    - nvm use 12.16.2
```
Comment (pacien) [inline .travis.yml:98]: Is it necessary to give a version number this precise?

Comment (OzoneGrif) [reply .travis.yml:98]: I prefer to be precise, some minor versions sometimes break things. Remember: This is Wawaswript

File: .travis.yml (hunk 2 above)
Comment (OzoneGrif) [reply .travis.yml:81]: right, oups

**Pull request closed without merging**